{{cookiecutter.project_slug}}/src/navigator/mainNavigator.js

```diff
@@ -13,9 +13,14 @@ const AppNavigator = createStackNavigator(
     SplashScreen: {
       screen: SplashScreen
     },
-    {% if cookiecutter.has_login_blueprint %}
+    {% if cookiecutter.has_login_blueprint == "True" %}
     EmailAuth: {
       screen: EmailAuthNavigator
+    },
+    {% endif %}
+    {% if cookiecutter.has_maps_blueprint == "True" %}
+    MapsScreen: {
+      screen: MapsScreen
```

Comment:
You need to create a `stackNavigator` instead of just rendering the component itself.

That way we can add more screens to your route., 

you can check the navigator inside EmailAuth to do yours 👍